evaporate.js

```diff
@@ -597,7 +597,6 @@
                         setStatus(COMPLETE);
                     } else {
                         l.d('headObject not found on S3.');
-                        me.name = awsKey;
```

Comment:
Ah, only some browsers with specific security settings (disabled) will provide the path to the file. It's explicitly disallowed on most browsers for security reasons. We just want to make sure that we don't see regressions when if we remove it.